### PR TITLE
fix: changed marshmallow-annotation version, temp solution

### PR DESCRIPTION
**Pull request by allisonsuarez**
### Summary of Changes

Using git+https://www.github.com/hilearn/marshmallow-annotations.git@a7a2dc96932430369bdef36555082df990ed9bef#egg=marshmallow-annotations as marshmallow-annotations version while waiting for dep to be maintained and compatible with marshmallow >3.0.

### CheckList

Make sure you have checked **all** steps below to ensure a timely review.

- [ ] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [ ] PR includes a summary of changes.

### CODE REVIEW

**Review by Golodhros — DISMISSED** [review]
LGTM

**Review by feng-tao — DISMISSED** [review]
do we need to bump the version of common as well?

File: setup.py [hunk 1]
```diff
@@ -31,8 +31,9 @@
         # long as they have a version of pyfoobar equal to or greater than 1.x
         # and less than 2.x installed.
         'flask>=1.0.2',
-        'marshmallow>=2.15.3,<3.0',
-        'marshmallow-annotations>=2.4.0,<3.0'
+        'marshmallow>=2.15.3,<=3.6',
+        ('git+https://www.github.com/hilearn/marshmallow-annotations.git@a7a2dc96932430369bd'
```
Comment (feng-tao) [inline setup.py:35]: do we need `(` here?

Comment (allisonsuarez) [reply setup.py:35]: I believe so because it is 1 logn string that I had to break up without having a newline

Comment (allisonsuarez) [reply setup.py:35]: I believe there would be an indentation error without the parenthesis

**Comment by allisonsuarez** [comment]
> do we need to bump the version of common as well?

Would this be a breaking change for any service on marshmallow < 3.0?

**Comment by verdan** [comment]
Slightly related to this, I have created RFC to introduce Python dataclasses instead of using attrs. 
Would be happy to have your feedback on that as well and if approved, should also keep that n mind while working on this change. 

https://github.com/amundsen-io/rfcs/pull/12

**Review by dikshathakur3119 — APPROVED** [review]

**Review by feng-tao — APPROVED** [review]

**Pull request merged**